kubetest/main.go

```diff
@@ -93,7 +93,7 @@ func defineFlags() *options {
 	flag.BoolVar(&o.charts, "charts", false, "If true, run charts tests")
 	flag.BoolVar(&o.checkSkew, "check-version-skew", true, "Verify client and server versions match")
 	flag.BoolVar(&o.checkLeaks, "check-leaked-resources", false, "Ensure project ends with the same resources")
-	flag.StringVar(&o.deployment, "deployment", "bash", "Choices: none/bash/kops/kubernetes-anywhere")
+	flag.StringVar(&o.deployment, "deployment", "bash", "Choices: none/bash/gke/kops/kubernetes-anywhere")
```

Comment:
It's a GKE deployment strategy - you could imagine using "--deployment=none --provider=gke" as well. I actually don't mind the explicitness of `--deployment=gke --provider=gke`, so I think I'll adopt your suggest just to verify that `--deployment=gke` requires `--provider=gke`.